doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -1477,6 +1477,11 @@ PSI Variables by Alpha
 
    A induction-classified SAPT term energy [Eh] implemented for SAPT2+3.
 
+.. psivar:: SAPT EXCH-IND30(S^INF) ENERGY
```

Comment:
Glad to see that nothing I did created problems here for you.

Reply:
All's well. The `-n` nitpicky option shows up missing glossary entries easily. Unfortunately docs weren't in a shape to use it during your excited state qcvars pass.